channel/src/test/java/com/synopsys/integration/alert/channel/jira/cloud/JiraCloudTestActionTest.java

```diff
@@ -149,7 +149,7 @@ private JiraCloudProperties createMockServerConfig() throws IssueTrackerExceptio
 
     private JiraIssueSearchProperties createSearchProperties() {
         return new JiraIssueSearchProperties("provider",
-            "providerUrl",
+            "providerURL",
```

Comment:
This would be a breaking change to the Alert Issue Property Indexer plugin, so we should leave this alone for now.